app/models/viewmodels/decoder/CHIEFErrors.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.viewmodels.decoder
+
+import com.github.tototoshi.csv.CSVReader
+import play.api.Logger
+
+import scala.io.Source
+
+case class CHIEFError(code: String, description: String)
+
+object CHIEFError {
```

Comment:
i know that `CHIEF` is acronym but having so much upper case letters in one place looks little awkward

Reply:
To be consistent with rest of the errors e.g. `ROEError` etc